### PR TITLE
fix: update itertools 0.8.0 -> 0.10.0

### PR DESCRIPTION
**Pull request by biomunky**
Targets https://github.com/uutils/coreutils/issues/2940

* since versions were mxing versions of x.y and x.y.z I changed all to x.y.z
* minor whitespace formatting

### CODE REVIEW

File: src/uu/printf/Cargo.toml [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-version = "0.0.12"
 authors = [
-    "Nathan Ross",
-    "uutils developers",
+  "Nathan Ross",
+  "uutils developers",
 ]
-license = "MIT"
 description = "printf ~ (uutils) FORMAT and display ARGUMENTS"
+license = "MIT"
```
Comment (sylvestre) [inline src/uu/printf/Cargo.toml:7]: I am a bit confused why you are moving stuff ? :)

Comment (biomunky) [reply src/uu/printf/Cargo.toml:7]: ah, that's not me, well not directly, it's the vscode format plugin for Cargo/Rust

Comment (sylvestre) [reply src/uu/printf/Cargo.toml:7]: if you don't mind, yeah :)

Comment (biomunky) [reply src/uu/printf/Cargo.toml:8]: I hope the whitespace fix-ups are acceptable

File: src/uu/printf/Cargo.toml [hunk 2]
```diff
@@ -18,10 +18,10 @@ edition = "2018"
...
 
 [dependencies]
-clap = { version = "3.0", features = ["wrap_help", "cargo"] }
-itertools = "0.8.0"
-uucore = { version=">=0.0.11", package="uucore", path="../../uucore", features=["memo"] }
-uucore_procs = { version=">=0.0.8", package="uucore_procs", path="../../uucore_procs" }
+clap = {version = "3.0", features = ["wrap_help", "cargo"]}
+itertools = "0.10.0"
+uucore = {version = ">=0.0.11", package = "uucore", path = "../../uucore", features = ["memo"]}
+uucore_procs = {version = ">=0.0.8", package = "uucore_procs", path = "../../uucore_procs"}
```
Comment (sylvestre) [inline src/uu/printf/Cargo.toml:24]: uucore_procs have been remove  :)

File: src/uu/printf/Cargo.toml [hunk 3]
```diff
@@ -18,10 +18,9 @@ edition = "2018"
 path = "src/printf.rs"
 
 [dependencies]
-clap = { version = "3.0", features = ["wrap_help", "cargo"] }
-itertools = "0.8.0"
-uucore = { version=">=0.0.11", package="uucore", path="../../uucore", features=["memo"] }
-uucore_procs = { version=">=0.0.8", package="uucore_procs", path="../../uucore_procs" }
+clap = {version = "3.0", features = ["wrap_help", "cargo"]}
```
Comment (sylvestre) [inline src/uu/printf/Cargo.toml:21]:
```suggestion
clap = { version = "3.0", features = ["wrap_help", "cargo"] }
```

File: src/uu/printf/Cargo.toml [hunk 4]
```diff
@@ -18,10 +18,9 @@ edition = "2018"
...
+itertools = "0.10.0"
+uucore = {version = ">=0.0.11", package = "uucore", path = "../../uucore", features = ["memo"]}
```
Comment (sylvestre) [inline src/uu/printf/Cargo.toml:23]:
```suggestion
uucore = { version = ">=0.0.11", package = "uucore", path = "../../uucore", features = ["memo"] }
```

**Review by sylvestre — COMMENTED** [review]
the rest of the configuration uses spaces :)
I value consistency :)

File: src/uu/printf/Cargo.toml (hunk 3 above)
Comment (biomunky) [reply src/uu/printf/Cargo.toml:21]: just fixing up

**Comment by biomunky** [comment]
@sylvestre - sorry, I closed the wrong PR, can you open. I think things should line up now.

**Comment by sylvestre** [comment]
seems that I can't reopen it ?!

**Comment by tertsdiepraam** [comment]
At first I couldn't either, but now it seems to be possible

**Comment by biomunky** [comment]
Nope, it's still there. I will have a look tomorrow - sorry folks, have taken more of your time than i should.

**Pull request closed without merging**